gammapy/scripts/download.py

```diff
@@ -0,0 +1,107 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Command line tool to download datasets and notebooks from gammapy-extra GitHub repo."""
+from __future__ import absolute_import, division, print_function, unicode_literals
+import logging
+import click
+
+import sys
+import requests
```

Comment:
How useful is `requests` here?

It's currently not used in Gammapy or installed as a dependency.
I.e. this is blocking this PR from going in:
https://ci.appveyor.com/project/cdeil/gammapy/build/1.0.4278/job/2fev6npu271cehix#L1360

If you can rewrite this to just use `urllib` without too much trouble, even if a little painful, please do.
Alternatively, `requests` must be added on the CI config files and setup.py and install docs as a required dependency.